src/exabgp/application/healthcheck.py

```diff
@@ -180,15 +181,15 @@ def syslog_address():
         logger.addHandler(ch)
 
 
-def loopback_ips(label, label_only):
+def loopback_ips(label, label_only, interface_numbering):
     """Retrieve loopback IP addresses"""
     logger.debug("Retrieve loopback IP addresses")
     addresses = []
 
     if sys.platform.startswith("linux"):
         # Use "ip" (ifconfig is not able to see all addresses)
         ipre = re.compile(r"^(?P<index>\d+):\s+(?P<name>\S+)\s+inet6?\s+" r"(?P<ip>[\da-f.:]+)/(?P<mask>\d+)\s+.*")
-        labelre = re.compile(r".*\s+lo:(?P<label>\S+).*")
+        labelre = re.compile(r".*lo:(?P<label>[^\\]+).*")
```

Comment:
This would match "hello" interface as well. Why change the regex?

Reply:
I wanted to drop the last `\` from the label. this is an example output in Linux:
```
1: lo    inet 127.0.0.1/8 scope host lo:label\       valid_lft forever preferred_lft forever
```
which the previous regex would still carry the `\`. I don't know what do you mean about the "hello" interface. Can you give me an example or suggest a regex for this case?

Reply:
Removing `\s+` opens the possibility to match more than `lo:something`. Use `[^\\\s]` to also include blanks, just in case the backslash is not always there.